applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h

```diff
@@ -138,30 +138,31 @@ class NodalValuesInterpolationProcess
     void Execute() override
     {
         // We create the locator
-        BinBasedFastPointLocator<TDim> PointLocator = BinBasedFastPointLocator<TDim>(mrOriginMainModelPart);
-        PointLocator.UpdateSearchDatabase();
+        BinBasedFastPointLocator<TDim> point_locator = BinBasedFastPointLocator<TDim>(mrOriginMainModelPart);
+        point_locator.UpdateSearchDatabase();
         
         // Iterate in the nodes
-        NodesArrayType& pNode = mrDestinationMainModelPart.Nodes();
-        auto numNodes = pNode.end() - pNode.begin();
+        NodesArrayType& p_node = mrDestinationMainModelPart.Nodes();
```

Comment:
pNode is a wrong name for this. It is not a pointer, but rather an array. Call it NodeContainer or something of the sort.

also  i wouldn't use auto in numNodes (it is a simple type), and i would call the function size()

     int numNodes = rDestinationMainModelPart.Nodes().size();

Reply:
Sorry, my fault